### PR TITLE
Yongbing chen/ui adjustment with designers feedback

### PR DESCRIPTION
**Pull request by yongbing-chen**
1. Move the confidence number to the right side pane
2. Have a check icon to show whether or not a field is revised
3. Have a before/after text (one more row?), add a button switch show/hide the text
4. Use sentence capitalization on flowing tool tip
5. Put “Actions”  in the “…”
6. Change the command text to “Auto-label multiple unlabeled documents”, and once user click it, we pop a dialog to set the batch size.

image

image

### CODE REVIEW

**Review by stew-ro — COMMENTED** [review]
We should use the filter button to expand into filter options for:
- tags on current document
- tags revised

Similar to MSFT Teams:
- image
- image

File: src/common/strings.ts [hunk 1]
```diff
@@ -449,7 +451,7 @@ export interface IAppStrings {
                             runOcrOnCurrentDocument: string,
                             runOcrOnAllDocuments: string,
                             runAutoLabelingCurrentDocument: string,
-                            runAutoLabelingOnNotLabelingDocuments: string,
+                            runAutoLabelingOnMultipleUnlabledDocuments: string,
```
Comment (starain-pactera) [inline src/common/strings.ts:454]: runAutoLabelingOnMultipleUnlabledDocuments  => runAutoLabelingOnMultiple**Unlabeld**Documents

File: src/common/localization/es-cl.ts [hunk 2]
```diff
@@ -455,7 +457,7 @@ export const spanish: IAppStrings = {
                             runOcrOnCurrentDocument: "Ejecutar OCR en el documento actual",
                             runOcrOnAllDocuments: "Ejecute OCR en todos los documentos",
                             runAutoLabelingCurrentDocument: "Etiquetar automáticamente el documento actual",
-                            runAutoLabelingOnNotLabelingDocuments: "Etiquetar automáticamente los siguientes ${batchSize} documentos sin etiquetar",
+                            runAutoLabelingOnMultipleUnlabledDocuments: "Etiquetar automáticamente varios documentos sin etiquetar",
```
Comment (starain-pactera) [inline src/common/localization/es-cl.ts:460]: runAutoLabelingOnMultipleUnlabledDocuments => runAutoLabelingOnMultipleUnlabeledDocuments

File: src/react/components/pages/editorPage/canvas.tsx [hunk 3]
```diff
@@ -258,7 +261,7 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
                     handleAssetDeleted={this.props.onAssetDeleted}
                     handleRunOcrForAllDocuments={this.runOcrForAllDocuments}
                     handleRunAutoLabelingOnCurrentDocument={this.runAutoLabelingOnCurrentDocument}
-                    handleRunAutoLabelingForRestDocuments={this.runAutoLabelingForRestDocuments}
+                    handleRunAutoLabelingOnMultipleUnlabledDocuments={this.runAutoLabelingOnMultipleUnlabledDocuments}
```
Comment (starain-pactera) [inline src/react/components/pages/editorPage/canvas.tsx:264]: handleRunAutoLabelingOnMultipleUnlabledDocuments => handleRunAutoLabelingOnMultipleUnlabeledDocuments
runAutoLabelingOnMultipleUnlabledDocuments  => runAutoLabelingOnMultipleUnlabeledDocuments

File: src/react/components/pages/editorPage/canvas.tsx [hunk 4]
```diff
@@ -362,6 +365,10 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
                         errorMessage: undefined,
                     })}
                 />
+                <BatchSizeModal
+                    ref={this.autoLabelingBatchSizeModal}
+                    onConfirm={this.confirmRunAutoLabelingOnMultipleUnlabledDocuments}
```
Comment (starain-pactera) [inline src/react/components/pages/editorPage/canvas.tsx:370]: confirmRunAutoLabelingOnMultipleUnlabledDocuments  => confirmRunAutoLabelingOnMultipleUnlabeledDocuments

File: src/react/components/pages/editorPage/canvas.tsx [hunk 5]
```diff
@@ -386,10 +393,13 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
             this.setAutoLabelingStatus(AutoLabelingStatus.done);
         }
     }
-    private runAutoLabelingForRestDocuments = async () => {
-        this.setState({ autoLableingStatus: AutoLabelingStatus.running });
-        await this.props.runAutoLabelingOnNextBatch();
-        this.setState({ autoLableingStatus: AutoLabelingStatus.done });
+    private runAutoLabelingOnMultipleUnlabledDocuments = async () => {
```
Comment (starain-pactera) [inline src/react/components/pages/editorPage/canvas.tsx:396]: runAutoLabelingOnMultipleUnlabledDocuments => runAutoLabelingOnMultipleUnlabeledDocuments

File: src/react/components/pages/editorPage/canvas.tsx [hunk 6]
```diff
@@ -386,10 +393,13 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
...
+        this.autoLabelingBatchSizeModal.current.openModal();
+    }
+    private confirmRunAutoLabelingOnMultipleUnlabledDocuments = async (batchSize: number) => {
```
Comment (starain-pactera) [inline src/react/components/pages/editorPage/canvas.tsx:399]: confirmRunAutoLabelingOnMultipleUnlabledDocuments => confirmRunAutoLabelingOnMultipleUnlabeledDocuments

File: src/react/components/pages/editorPage/canvasCommandBar.tsx [hunk 7]
```diff
@@ -15,7 +15,7 @@ interface ICanvasCommandBarProps {
     handleRunOcr?: () => void;
     handleRunOcrForAllDocuments?: () => void;
     handleRunAutoLabelingOnCurrentDocument?: () => void;
-    handleRunAutoLabelingForRestDocuments?: () => void;
+    handleRunAutoLabelingOnMultipleUnlabledDocuments?: () => void;
```
Comment (starain-pactera) [inline src/react/components/pages/editorPage/canvasCommandBar.tsx:18]: handleRunAutoLabelingOnMultipleUnlabledDocuments => handleRunAutoLabelingOnMultipleUnlabeledDocuments

File: src/react/components/pages/editorPage/canvasCommandBar.tsx [hunk 8]
```diff
@@ -188,14 +186,14 @@ export const CanvasCommandBar: React.FunctionComponent<ICanvasCommandBarProps> =
                         },
                     },
                     {
-                        key: "runAutoLabelingForRestDocuments",
-                        text: interpolate(strings.editorPage.canvas.canvasCommandBar.farItems.additionalActions.subIMenuItems.runAutoLabelingOnNotLabelingDocuments, { batchSize: constants.autoLabelBatchSize }),
+                        key: "runAutoLabelingOnMultipleUnlabledDocuments",
```
Comment (starain-pactera) [inline src/react/components/pages/editorPage/canvasCommandBar.tsx:189]: runAutoLabelingOnMultipleUnlabledDocuments => runAutoLabelingOnMultipleUnlabeledDocuments

File: src/react/components/pages/editorPage/canvasCommandBar.tsx [hunk 9]
```diff
@@ -188,14 +186,14 @@ export const CanvasCommandBar: React.FunctionComponent<ICanvasCommandBarProps> =
...
+                        text: strings.editorPage.canvas.canvasCommandBar.farItems.additionalActions.subIMenuItems.runAutoLabelingOnMultipleUnlabledDocuments,
```
Comment (starain-pactera) [inline src/react/components/pages/editorPage/canvasCommandBar.tsx:190]: runAutoLabelingOnMultipleUnlabeledDocuments

File: src/react/components/pages/editorPage/canvasCommandBar.tsx [hunk 10]
```diff
@@ -188,14 +186,14 @@ export const CanvasCommandBar: React.FunctionComponent<ICanvasCommandBarProps> =
...
                         iconProps: { iconName: "Tag" },
                         disabled: disableAutoLabeling,
                         title: props.project.predictModelId ? "" :
                             strings.editorPage.canvas.canvasCommandBar.farItems.additionalActions.subIMenuItems.noPredictModelOnProject,
                         onClick: () => {
-                            if (props.handleRunAutoLabelingForRestDocuments) props.handleRunAutoLabelingForRestDocuments();
+                            if (props.handleRunAutoLabelingOnMultipleUnlabledDocuments) props.handleRunAutoLabelingOnMultipleUnlabledDocuments();
```
Comment (starain-pactera) [inline src/react/components/pages/editorPage/canvasCommandBar.tsx:196]: handleRunAutoLabelingOnMultipleUnlabledDocuments => handleRunAutoLabelingOnMultipleUnlabeledDocuments

**Review by starain-pactera — COMMENTED** [review]
Some words are incorrect. I added comments for each line.

**Comment by starain-pactera** [comment]
For microsoft.OCR-Form-Tools pipeline issue, **showOriginLabels** property should be added to **createProps** function. Related file: src/react/components/common/tagInput/tagInputItem.test.tsx.

File: src/common/localization/en-us.ts [hunk 11]
```diff
@@ -454,7 +456,7 @@ export const english: IAppStrings = {
                             runOcrOnCurrentDocument: "Run OCR on current document",
                             runOcrOnAllDocuments: "Run OCR on all documents",
                             runAutoLabelingCurrentDocument: "Auto-label the current document",
-                            runAutoLabelingOnNotLabelingDocuments: "Auto-label next ${batchSize} unlabeled documents",
+                            runAutoLabelingOnMultipleUnlabledDocuments: "Auto-label multiple unlabeled documents",
```
Comment (alex-krasn) [inline src/common/localization/en-us.ts:459]: The right spelling is runAutoLabelingOnMultiple**Unlabeled**Documents

**Comment by starain-pactera** [comment]
UI changed:  
image

**Review by starain-pactera — APPROVED** [review]

**Review by stew-ro — COMMENTED** [review]
There's still some missing designer feedback. Like changing the document state icons. Will that be included in another PR?

**Comment by starain-pactera** [comment]
Hello stew-ro,
Yes, we will include them in another PR. 
Thanks

**Pull request merged**